doc/changelog.rst

```diff
@@ -1,6 +1,7 @@
 Changelog
 =========
 
+* :feature:`633` Allow retrieving exception object from `handling_exeptions` context
```

Comment:
When using the `handling_exceptions`, it is now possible to obtain the exception object that was handled